### DISCUSSION TITLE
也许我们可以弄一个discord来交流

### DISCUSSION THREAD

**Discussion by habaohaba** [PKUFlyingPig/cs-self-learning#536] [General]
1. qq群不好用
2. 各个群太过分散
3. 我感觉15445官方的非cmu学生的discord氛围挺好，也能够进行分区，各种功能也有提供

**Comment by shuodotsun** [comment]
你说的哪个？
我这有个discord，欢迎加入，


https://discord.gg/57kYCeSkCb

cs-diy-self learning:

Discord server: cs diy self learning.
欢迎👏
一起学习讨论

**Reply by habaohaba** [reply]
这个链接打不开了啊。

**Reply by shuodotsun** [reply]
谢谢回复。这个最新的

https://discord.gg/57kYCeSkCb
